data/src/main/scala/cats/data/Or.scala

```diff
@@ -381,6 +374,10 @@ sealed abstract class OrInstances extends OrInstances0 {
       def empty =
         ROr(Monoid[B].empty)
     }
+
+  implicit def OrdShow[A: Show, B: Show]: Show[A Or B] =
```

Comment:
Oh sorry one more thing that cats probably should develop a convention for:

In `oneAndShow` the type class instances are referenced as implicit parameters while in `OrdShow` it uses `Show[A]` to grab the instance. It would be nice to be consistent about this. My vote would be for the `oneAndShow` approach when the instance needs to be referenced but for the `OrdShow` approach when the evidence is just needed so it can be passed along to another function call. I'm curious what others think about this.